jobspec/parse_group.go

```diff
@@ -56,6 +56,7 @@ func parseGroups(result *api.Job, list *ast.ObjectList) error {
 			"service",
 			"volume",
 			"scaling",
+			"stop_after_client_disconnect",
```

Comment:
My understanding is this effectively won't be exposed until we do the server-side work? So it should be ok to delay documenting this until that PR?

Reply:
yes, I pulled it out of the api